### ISSUE TITLE
You stole my project's name :D

### ISSUE THREAD

**Issue by floordiv** [sigmavirus24/rush#16]
I'm joking, your repository is older than mine. But still, how can I publish my [webserver](https://github.com/floordiv/rush) on PyPI now?..

**Comment by sigmavirus24** [comment]
> But still, how can I publish my [webserver](https://github.com/floordiv/rush) on PyPI now?

You could choose a different name, or "namespace" it (e.g., `floordiv.rush`). Based on your description it's an experiment

**Comment by sigmavirus24** [comment]
A friend suggested you might be hoping I'd give up the name. I know there to be companies relying on this library and doing that would be a horrible experience for them because there's no way to signal to them "Hey, this isn't what you want any longer"

**Comment by floordiv** [comment]
> A friend suggested you might be hoping I'd give up the name. I know there to be companies relying on this library and doing that would be a horrible experience for them because there's no way to signal to them "Hey, this isn't what you want any longer"

Of course no. It is awful cause deleting/changing library's name on PyPI is a thing that can bring a lot of problems. I just found out that my project's name isn't unique, and that surprised me. Thank you for telling me about namespaces! I really didn't know about them. Anyway, I decided to name the repository on PyPI as a "rush-webserver" - this will be better, I think.

About experiment - yes, you noted it correctly :D Anyway I wanna try to distribute it as something serious to get some new expierence about projects' distribution
